### PR TITLE
Making the doc more formal

### PR DESCRIPTION
**Pull request by chfzdn**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
+# Contributing
```
Comment (sabieber) [inline CONTRIBUTING.md:1]: A little extensive in my eyes ... also dont think we have to copy other peoples docs

File: CREDITS [hunk 2]
```diff
@@ -1,9 +1,11 @@
-DEVELOPMENT
+PROJECT LEADER
```
Comment (sabieber) [inline CREDITS:1]: PROJECT OWNER?

File: CREDITS [hunk 3]
```diff
@@ -18,6 +20,7 @@ hostel by Ed Piel from the Noun Project
 market by arif fajar yulianto from the Noun Project
 Factory by BGBOXXX Design from the Noun Project
 
-GENERAL
+Thanks to all contributors for your help.
+Thanks to @sabieber for fire depts, many insights and code fixes.
```
Comment (sabieber) [inline CREDITS:24]: We can leave that one out because of the line above ;)

File: README.md [hunk 4]
```diff
@@ -4,43 +4,42 @@
...
 ![KotCity Screenshot](screenshot.jpg?raw=true "Screenshot of the game's UI and an example city")
 
-## Gimme the Software!
+## New in this version
 
-A build for Windows, macOS and Linux is available at https://github.com/kotcity/kotcity/releases/tag/0.45
+95% of these are from my main man @sabieber, who is the KotCity hero.
 
-Java 8+ is required. On Windows it will look for JRE and bring you to download page if you don't have it.
+See our roadmap at https://github.com/kotcity/kotcity/wiki/Roadmap
```
Comment (sabieber) [inline README.md:15]: Roadmap should be mentioned in the next up section

File: README.md [hunk 5]
```diff
@@ -4,43 +4,42 @@
...
+* Buildings construct a little more often.
+* Tweaked required kots (equivalent to *sims*) for industrial plant.
+* Minimal map size reduced (you can start a 32*32 map).
+* Right click and drag to pan map.
+* Update cursor on map pan.
+* Dynamic framerate adjustment during panning & zooming.
+* Zoom to mouse position on zoom.
 
-## New in this version
+## What is KotCity?
```
Comment (sabieber) [inline README.md:26]: Why all the moving around again? I think having the link to the release above all the text is a lot better

File: README.md [hunk 6]
```diff
@@ -61,52 +60,17 @@ The UI is done with FXML created with [Gluon's SceneBuilder](http://gluonhq.com/
 ### Q: Why Kotlin?  
 **A:** It has a lot of libraries (pick any random Java library...) It's pretty productive! Gee-whiz functional stuff baked in. Besides, if this project gets to a place where it's really awesome but just needs extra speed we can reach for that C++ or Rust book.
 
-### Q: Bedeutet "Kot" nicht 💩 in Deutsch?
```
Comment (sabieber) [inline README.md:64]: Why would you kill that? :D

**Review by sabieber — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (chfzdn) [reply CONTRIBUTING.md:1]: As long as credited, it's fine. Feel free to express yours.

**Comment by chfzdn** [comment]
Due to restructurisation to my fork, this PR is closed.

File: README.md (hunk 6 above)
Comment (kotcity) [reply README.md:64]: We definitely gotta leave it. It _really is_ a FAQ!

**Pull request closed without merging**